src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1966,6 +1966,63 @@ bool isCondValueInContextElementResponse(ConditionValueList* condValues, Context
 }
 
 
+
+/* ****************************************************************************
+*
+* someEmptyCondValue -
+*
+* This logic would be MUCH simpler in the case conditions was a single field instead of a vector.
```

Comment:
Yes, why don't? In the future, when ONTIMEINTERVAL gets removed the data model for csubs document could be simplified, to avoid the conditions vector. Thus, I think the comment is relevant and should be kept.